index.bs

```diff
@@ -128,7 +140,25 @@ This callback will be invoked when there are changes to <i>target</i>'s intersec
     </pre>
 
     <div dfn-type="method" dfn-for="IntersectionObserver">
-        : <dfn>observe()</dfn>
+        : <dfn>Constructor(callback, options)</dfn>
```

Comment:
Whatever you want to display. The `lt` overrides the linking text, so you can put whatever you want as the actual content.  What Domenic suggested is fine.